Index: lab1/.env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>API_KEY = 'live_xJowRcD2GFmlEhuAjQkdZhH4axl47FGCfI1JL2KHRs1HgYyGUmt4y7GxV7NvLufg'\r\nAPI_URL = 'https://api.thecatapi.com/v1/images/search'\r\nHEADERS = {'x-api-key': API_KEY}
===================================================================
diff --git a/lab1/.env b/lab1/.env
--- a/lab1/.env	(revision 809f8e23573fd55e5fbcb2a8aad034dd9123d2d9)
+++ b/lab1/.env	(date 1743063027334)
@@ -1,3 +1,4 @@
 API_KEY = 'live_xJowRcD2GFmlEhuAjQkdZhH4axl47FGCfI1JL2KHRs1HgYyGUmt4y7GxV7NvLufg'
 API_URL = 'https://api.thecatapi.com/v1/images/search'
-HEADERS = {'x-api-key': API_KEY}
\ No newline at end of file
+HEADERS = {'x-api-key': API_KEY}
+PARAMS = {'has_breeds': 1, 'limit': 1}
\ No newline at end of file
